Developer-Guide.md

```diff
@@ -285,7 +295,7 @@ $ sudo systemctl daemon-reload
 $ sudo systemctl restart docker
 ```
 
-# Create a Kata Container
+## Create a Kata Containers container
```

Comment:
'Containers container' just reads strange :-) I suspect we can just use `Create a Kata Container`.
An additional nicety might be to add something like a `$ uname -a` once the container is up to visually show/acknowledge we are in a Kata Container, but I don't know if this script is executed as part of the 'execute documentation' system, and hence if that is really feasible.

Reply:
Well I understand but it is either we say `Kata Containers container` because we consider `Kata Containers` as a brand/project name, or we say `Kata container` for simplicity, but I don't like `Kata Container` because upper case for both `Kata` and `Container` looks like we butchered the project name here.